node/core/approval-voting/src/lib.rs

```diff
@@ -592,6 +595,10 @@ async fn handle_approved_ancestor(
 
 	if target_number <= lower_bound { return Ok(None) }
 
+	span.add_stage(JaegerStage::ApprovalChecking);
+	span.add_string_tag("target-number", &format!("{}", target_number));
```

Comment:
I'll do another sweep and see if we can use smth like `add_tag(&str, impl ToString)` and another for `fmt::Debug` impl. That would allow easier verification of consistent formatting and get rid of a lot of `&format!(..)` macros.